tests/Integration/inc/functions/rocketIsLiveSite.php

```diff
@@ -44,6 +45,8 @@ public function testShouldReturnFalseWhenLocalOrStaging() {
 			$this->assertFalse( rocket_is_live_site() );
 			remove_filter( 'home_url', $callback );
 		}
+
+		remove_filter('rocket_staging_list', [$this, 'getLocalStagingSites']);
```

Comment:
```suggestion
		remove_filter('rocket_staging_list', [$this, 'getLocalStagingDomains']);
```

I think u mean `getLocalStagingDomains` here, correct?

Also, Are you sure that this will not send the request to our SaaS to get the staging domains with the filter `rocket_staging_list`?

Reply:
Yea that's what I meant, so I corrected it. 🙏

Reply:
If you use the filter, you'll get to the saas to get the list, however if you don't you won't. I'm not sure to fully understand your question :x

Reply:
I mean going to SaaS at this test is useless because we already added a callback to the filter to get the response from ur method `getLocalStagingDomains` 
It's a minor change but wanted to discuss it with u, I prefer mocking the response from the API instead of adding a new callback to cover also that the request is being sent but not a real request (because we will mock it before the external request happens)
what do u think?
I'll approve the PR but let's discuss that.